components/Input/ColorInput/index.js

```diff
@@ -94,26 +110,37 @@ const defaultProps = {
     ],
 };
 
-class ColorInput extends React.PureComponent {
-    static propTypes = propTypes;
+function ColorInput(props) {
+    const {
+        onChange,
+        showHintAndError,
+        label,
+        className,
+        value,
+        showLabel,
+        error,
+        hint,
+        disabled,
+        readOnly,
+        persistentHintAndError,
+        type,
+        colors,
+        showSwatches,
+    } = props;
 
-    static defaultProps = defaultProps;
+    const [showColorPicker, setShowColorPicker] = useState(false);
+    const [boundingClientRect, setBoundingClientRect] = useState({});
+    const containerRef = useRef(undefined);
+    const [selectedScheme, setSelectedScheme] = useState('sequential');
+    // NOTE: In color brewer each scheme group has at least one color swatch with 8 items
+    const [selectedNumberOfColors, setSelectedNumberOfColors] = useState('8');
```

Comment:
Value for selectedNumberOfColors is string but the value from options is a number